src/json.rs

```diff
@@ -199,6 +199,42 @@
 //!     let decoded: TestStruct = json::decode(json_str.as_slice()).unwrap();
 //! }
 //! ```
+//!
+//! ## Parsing a `str` to `Json` and reading the result
+//!
+//! ```rust
+//! # #![allow(unstable)]
+//! extern crate "rustc-serialize" as rustc_serialize;
+//! use rustc_serialize::json::Json;
+//!
+//! fn main() {
+//!     let data = Json::from_str("{\"foo\": 13, \"bar\": \"baz\"}").unwrap();
+//!     println!("data: {}", data);
+//!     // data: {"bar":"baz","foo":13}
+//!     println!("object? {}", data.is_object());
+//!     // object? true
+//!
+//!     let obj = data.as_object().unwrap();
+//!     let foo = obj.get("foo").unwrap();
+//!     println!("array? {}", foo.is_array());
```

Comment:
Yep, the above `is_object` example is enough to let people know about the `is_*` methods.